### DISCUSSION TITLE
Can't get Tailwind and friendly-snippets suggestions in htmldjango

### DISCUSSION THREAD

**Discussion by lcfd** [hrsh7th/nvim-cmp#1253] [Q&A]
Hi! 👋 

I've been trying to make suggestions work in `htmldjango` templates for days.
Tailwind works in `html` files.
In `htmldjango` I can only get the pure `html` snippets to work.

Can anyone figure out what's going on?
Save me from madness, pls 😆 

**MY CONFIG**

https://github.com/lcfd/NaryaVim

**Comment by fitrh** [comment]
Do you see `tailwindcss` when running `:LspInfo` within a `djangohtml` file?

I don't work with djangohtml on a daily basis, but using some of my old projects,
tailwindcss seems to be working fine, so I believe it's an LSP issue rather than
cmp itself

![20221021_201111](https://user-images.githubusercontent.com/55179750/197196229-4a349e1e-d073-4b39-b051-c4c581f5b42f.jpeg)

**Reply by lcfd** [reply]
This is what I see using `:LspInfo`.

![CleanShot 2022-10-21 at 14 47 02](https://user-images.githubusercontent.com/9001053/197199287-23a7a7a5-d3a4-4966-93b1-2085f068735e.png)

tailwindcss client is attached, that's why is strange.

What seems more strange to me is not having the other snippets. Like `ifs`, or `includes`.
They should work anyway because of the `htmldjango` recognition.

**Reply by lcfd** [reply]
UPDATE: I managed to make Django snippets works. There is an error in friendly-snippets. I have already created a PR.

Still no TailwindCSS (it's working even in Astro 😆).
